### ISSUE TITLE
Handle second second screen orientation

### ISSUE THREAD

**Issue by pintaf** [maruos/maruos#157]
Second screens are handled by Android in maru as far as I understood.

How can we set orientation of the second screen ? for example if it is in portrait mode.

Should it be done by the linux container or handled by Android ?
If it is the later, here are some hints I found:

https://play.google.com/store/apps/details?id=com.farmerbb.secondscreen.free&hl=en_US&gl=US&showAllReviews=true
https://android.stackexchange.com/questions/117003/allow-rotate-into-portrait-mode-but-do-not-rotate-external-display
https://stackoverflow.com/questions/54502032/how-can-we-programmatically-control-the-screen-orientation-in-secondary-displays

**Comment by utzcoz** [comment]
Yes, we can do it. We just need to change the surface size used by `mflinger`, and the `mclient` will adapt to it.
@pintaf could you provide the use case for this feature? If it may be used for some important occasion, we could consider to implement it.

**Comment by pintaf** [comment]
no, I would not say it's such an important occasion, but maybe if we allow setting the surface size (that sometimes default to 720p) in the settings, maybe we could also add a setting for portrait or landscape mode
